### PR TITLE
create integration test for BPMNFormModelGenerator

### PR DESCRIPTION
**Pull request by jsoltes**
Tests generateProcessFormModel, generateTaskFormModels and generateTaskFormModel methods on two valid process definitions loaded from resources.
Part of this commit is also cleanup of the unused private class TaskVariableSetting from the BPMNFormModelGenerator.

@pefernan, @manstis could you please review?
@pefernan, there are two tests ignored because of the JBPM-5977. Could you please unignore them later in the pull-request fixing the bug?

### CODE REVIEW

File: kie-wb-common-forms/kie-wb-common-forms-integrations/kie-wb-common-forms-jbpm-integration/kie-wb-common-forms-jbpm-integration-backend/src/test/java/org/kie/workbench/common/forms/jbpm/server/service/impl/BPMNFormModelGeneratorImplTest.java [hunk 1]
```diff
@@ -0,0 +1,259 @@
+package org.kie.workbench.common.forms.jbpm.server.service.impl;
```
Comment (manstis) [inline kie-wb-common-forms/kie-wb-common-forms-integrations/kie-wb-common-forms-jbpm-integration/kie-wb-common-forms-jbpm-integration-backend/src/test/java/org/kie/workbench/common/forms/jbpm/server/service/impl/BPMNFormModelGeneratorImplTest.java]: License please

Comment (jsoltes) [inline kie-wb-common-forms/kie-wb-common-forms-integrations/kie-wb-common-forms-jbpm-integration/kie-wb-common-forms-jbpm-integration-backend/src/test/java/org/kie/workbench/common/forms/jbpm/server/service/impl/BPMNFormModelGeneratorImplTest.java]: oh, man! I always forget this, thank you @manstis!

**Comment by jsoltes** [comment]
Looks like recent pull-request (https://github.com/AppFormer/uberfire/pull/694) is making the kie-wb-common build fail. I wrote a comment there, so we need to retest this after that pr is reverted.

**Comment by jsoltes** [comment]
jenkins retest this

**Comment by jsoltes** [comment]
jenkins failure seems to be unrelated, jenkins retest this

**Comment by ederign** [comment]
The AppFormer commit was reverted: https://github.com/AppFormer/uberfire/commit/b7235b0cb287ea37798bfb21556bbc6661d6074e

**Comment by ederign** [comment]
jenkins retest this

**Pull request merged**